utilities/hooks/useBreakpointSize.ts

```diff
@@ -0,0 +1,24 @@
+import { useState, useEffect } from 'react'
+
+export const MOBILE_BREAKPOINT = 'mobile'
```

Comment:
where do you use these ? just curious

Reply:
Lines 8 and 13 directly in this file. Then `MOBILE_BREAKPOINT` is used in nav and transition components